cherrypy/lib/sessions.py

```diff
@@ -603,6 +603,10 @@ def clean_up(self):
                         if expiration_time < now:
                             # Session expired: deleting it
                             os.unlink(path)
+
+                except pickle.UnpicklingError:
```

Comment:
Could you add a test for this?

Reply:
I wish I am able to write a test for this - but with limited skill, I couldn't figure out how to write one - if possible could you please provide some guidance and thoughts?

Reply:
I'm not sure either. It would probably be an integration test that sets up a server with sessions, makes a query (which results in a session file appearing on disk), then corrupts that file by writing arbitrary data into it (a random byte might be enough), followed by tearing down the server and that should raise the exception. Maybe, it'd be easier (or more stable) with calling `clean_up()` and checking that the file disappeared from disk. Additionally, it would be wise to use `pytest-mocker`'s `mocker.spy()` on the `_load()` method to verify that the exception was raised, indeed.